client/templates/dashboard/dashboard.js

```diff
@@ -39,5 +42,14 @@ Template.dashboardShopifyProducts.events({
     } else {
       Meteor.call('importShopifyProducts/importProducts', update, productType);
     }
+  },
+  'submit #import-products-csv-form': function (event) {
+    event.preventDefault();
+    Papa.parse(event.target.csvImportProductsFile.files[0], {
+      header: true,
+      complete: function (results) {
+        Meteor.call('importCSVProducts/importProducts', results.data);
+      }
```

Comment:
Good catch. I should add an error handler in here. There's 2 ways to do it in Papa Parse - with an `error:` option and with an `error` object on the `results object` - I should add both.